src/opers.c

```diff
@@ -3389,12 +3390,15 @@ static Obj DoSetterFunction(Obj self, Obj obj, Obj value)
     flag2  = INT_INTOBJ( FLAG2_FILT(tester) );
     type   = TYPE_OBJ_FEO(obj);
     flags  = FLAGS_TYPE(type);
+
+    UInt rnam = (UInt)INT_INTOBJ(ELM_PLIST(tmp,1));
+
     if ( SAFE_C_ELM_FLAGS(flags,flag2) ) {
+        CALL_3ARGS(CHECK_REPEATED_ATTRIBUTE_SET, obj, NAME_RNAM(rnam), value);
```

Comment:
Just wondering: This code isn't run for mutable attributes, is it?

Reply:
Nope, they are handled in a different function (but I'll add a test, just to demonstrate this)